.gitignore

```diff
@@ -21,3 +21,6 @@ _output/
 
 # OSX trash
 .DS_Store
+
+# sub chart tgz
+charts/karmada/charts
```

Comment:
Should we remove the path `https://github.com/karmada-io/karmada/tree/master/charts/karmada/charts` ? it's already in the repo.

Reply:
This is the package we depency on, which is a tgz file. When we pack chart, we can exec `helm package ./charts/karmada --dependency-update` to automatically download it for us.
I think we should not upload tgz file to our repo.

Reply:
we can delete the file `https://github.com/karmada-io/karmada/tree/master/charts/karmada/charts` together.

Reply:
Yeah, it has been already in our repo. Can you delete it together in this pr?

Reply:
done

Reply:
Hi @calvin0327, could you help check why chart-lint failed?